### PR TITLE
Feature/setup terraform

### PR DESCRIPTION
**Pull request by JDeBo**
- Set up initial skeleton instances (User data to come)
- Added security groups
- Added outputs for ips (need to get public IPs configured with these modules still)
``` 
instance_ips = {
  "testing-osc-workshop-linux-team1" = ""
  "testing-osc-workshop-linux-team2" = ""
}
```
- Validated everything deploys successfully

### CODE REVIEW

File: linux/terraform/main.tf [hunk 1]
```diff
@@ -7,16 +7,17 @@ provider "aws" {
 locals {
   db_ip  = "10.0.1.10"
   region = var.aws_region
-
+  name = "${var.event}-osc-workshop-linux"
   tags = {
-
+    event = var.event
```
Comment (ryapric) [inline linux/terraform/main.tf:12]: who do you think i am

File: linux/terraform/main.tf [hunk 2]
```diff
@@ -7,16 +7,17 @@ provider "aws" {
...
+    does_ryan_check_prs = false
```
Comment (ryapric) [inline linux/terraform/main.tf:13]: who do you think i am

Comment (JDeBo) [reply linux/terraform/main.tf]: Test passed

**Comment by JDeBo** [comment]
provision-ec2 script now appears to be working fully. Note: Left the security group rules as-is for now, which works great with a static public IP, may break again in a location with a variable IP.
image

**Comment by JDeBo** [comment]
Added an outline for the messed up git repo module. Need to decide if we really like that or not.

File: .gitignore [hunk 3]
```diff
@@ -1 +1,2 @@
 .vagrant
+.DS_Store
```
Comment (ryapric) [inline .gitignore:2]: or just DON'T USE A MAC

File: linux/scripts/init.sh [hunk 4]
```diff
@@ -85,12 +90,23 @@ printf 'All done!\n'
...
+  git config user.email "bugs@bigbadbunnies.com"
+  git remote add origin git@github.com/bigbadbunnies/carrot-cruncher
+  git add .
+  git commit -m "WIP"
+  git branch release/bunnies_v1 && git checkout release/bunnies_v1
+  sed -i -e 's/printing/picking/g' -e 's/money/carrots/g' -e 's/CHA-CHING/CRUNCH/g' main.go
+  git add .
+  git commit -m "Success"
+  git checkout main
+fi
```
Comment (ryapric) [inline linux/scripts/init.sh:109]: I'm gonna pull this out into its own branch, so that we can get the Terraform code in as-is.

Comment (ryapric) [reply linux/scripts/init.sh]: Pulled out to `feature/add-git-scoring-step`

**Review by ryapric — APPROVED** [review]
Changed a few things but overall LGTM, THANK YOU SO MUCH

**Pull request merged**